docs/guides/all-keys-one-secret.md

```diff
@@ -18,10 +18,60 @@ Now, when creating our ExternalSecret resource, instead of using the data field,
 ```yaml
 {% include 'gcpsm-data-from-external-secret.yaml' %}
 ```
+<br>
+Here, "example" is the name of Kubernetes secrets which will be created.
+<br>
+when we use 
 
-To check both values we can run:
+```
+  dataFrom:
+  - extract:
+      key: all-keys-example-secret
+```
+We get all the secrets present in the key-value pair in the secret manager(external secret store of GCP or AWS or Azure) in the form of key values and also we can pass either all or a few key-values as environment variables.
```

Comment:
I don't think this description makes it clearer. 

"All secrets"? Are you implying each key value inside the secret of the external provider is a secret? That's very confusing

Reply:
Added a more proper explanation.